### PR TITLE
Adds pyuoi recipe

### PR DESCRIPTION
**Pull request by JesseLivezey**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by pssachdeva** [comment]
I agree to be a maintainer.

**Comment by ajtritt** [comment]
I agree to be a maintainer.

File: recipes/pyuoi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - h5py
+    - scikit-learn
+
+test:
+  imports:
+    - pyuoi
+
+about:
+  home: https://github.com/BouchardLab/pyuoi
+  license: BSD
```
Comment (chrisburr) [inline recipes/pyuoi/meta.yaml]: Which BSD licence is it? The most common is this but please check 
```suggestion
  license: BSD-3-Clause
```

File: recipes/pyuoi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - numpy
```
Comment (chrisburr) [inline recipes/pyuoi/meta.yaml]: Do you need numpy here? You probably only need it if you’re using the C API and, if you are, check the conda forge docs for how to pin it at runtime.

Comment (JesseLivezey) [reply recipes/pyuoi/meta.yaml]: We build against numpy and I've updated things to what I believe is correct from here
https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

**Pull request merged**